addons/website_sale/static/src/js/components/media_dialog/kanban_record.scss

```
@@ -0,0 +1,9 @@
+.o_field_website_sale_x2_many_field .o_kanban_renderer {
+    --KanbanRecord-width: 100px;
+
+    .o_field_website_sale_image_field img {
+        height: 128px;
+        width: 168.86px;
+        object-fit: contain;
+    }
+}
```

```suggestion
.o_form_renderer { 
  .o_field_website_sale_x2_many_field .o_kanban_renderer {
      --KanbanRecord-width: 100px;
  
      .o_field_website_sale_image_field img {
          height: 128px;
          width: 168.86px;
          object-fit: contain;
      }
  }
```

I am afraid  about fix value. please check in all devices